docs/section4/4-3.md

```diff
@@ -0,0 +1,46 @@
+# ASHVAC – Rule 4-3
+
+**Rule ID:** 4-3
+ 
+**Rule Description:** HVAC Fan Schedules. Schedules for HVAC fans that provide outdoor air for ventilation shall run continuously whenever spaces are occupied.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules for the proposed building  
+**Appendix G Section Reference:**  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** 
+
+1. Not applicable for for heating or cooling system is being simulated only to meet the requirements described in Table G3.1.
+
```

Comment:
This is TBD, I think, because right now this RDS only applies to occupied hours so those other exceptions would not be relevant during occ hours. Correct?

Reply:
@mkarpman and @claperle I'm not sure what we have the applicability checks (2) and (3)
2. P-RMR airside HVAC system are been modeled correctly per Table G3.1 #10 proposed c: `if rule 10-10 == "PASS"`
3. P-RMR airside HVAC system are been modeled correctly per Table G3.1 #10 proposed d: `if rule 10-11 == "PASS"`

10-10 states - Where no heating system exists or no heating system has been submitted with design documents, the proposed building system type shall be the same system as modeled in the baseline building design and shall comply with but not exceed the requirements of Section 6

10-11 Except for spaces with baseline system 9 or 10, if no cooling system exists or no cooling system has been submitted with design documents, the proposed building cooling system type shall be the same as modeled in the baseline building design and shall comply with the requirements of Section 6.

Not sure what 10-10 and 10-11 need to pass as an applicability check for this rule.